src/components/DragonsCreator.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove this line of code because it is hindering prop checks. PropTypes ensures that the passed value is of the correct data type and helps in preventing unexpected behavior. 😎️

Reply:
done